### ISSUE TITLE
black screen and crash - Unable to recover from a kernel exception. The application must close. Error 3 (NVIDIA OpenGL Driver)

### ISSUE THREAD

**Issue by bobokox** [alicevision/meshroom#229]
Hey!

First of all, very nice programm, I really love it!
But I have a problem. Meshroom crashes randomly while the progress.
I have a new graphics card since a few days. It's the GTX 1080ti. First I had the same error while rendering in blender too, but I was able to fix it with the right settings. In Meshroom, I can't find settings releated to the gpu.

I'm also in contact with the nvidia support. But maybe there's a possibility to solve this in meshroom directly?

The log in the Win10 Event Viewer (NVIDIA OpenGL Driver):
```
Unable to recover from a kernel exception. The application must close.
Error code: 3 (subcode 2)
 (pid=1240 tid=11176 meshroom.exe 64bit)
```

And the log from the applications section:
`Name der fehlerhaften Anwendung: Meshroom.exe, Version: 2018.1.0.0, Zeitstempel: 0x5a355fd7
Name des fehlerhaften Moduls: nvoglv64.dll, Version: 24.21.13.9907, Zeitstempel: 0x5b7be39a
Ausnahmecode: 0xc0000409
Fehleroffset: 0x0000000000dbad59
ID des fehlerhaften Prozesses: 0x4d8
Startzeit der fehlerhaften Anwendung: 0x01d442189e5925e3
Pfad der fehlerhaften Anwendung: C:\Programme\Meshroom-2018.1.0\Meshroom.exe
Pfad des fehlerhaften Moduls: C:\WINDOWS\System32\DriverStore\FileRepository\nv_dispi.inf_amd64_f5be1f8d25335236\nvoglv64.dll
Berichtskennung: f357c045-32fe-422a-905a-4e175c507d28`

Sry that it is in german. hope it's understandable.

My System:
Win 10 Pro
ga990xa-ud3
AMD FX-9590 8-Core Processor
16 GB Ram
NVidia GFX 1080ti

Thank you for your help!

**Comment by sirwillard** [comment]
Your 1080Ti should have worked in blender right out of the box, so that is already strange.  What settings did you change? That would be helpful.

**Comment by sirwillard** [comment]
Oh wow, I didn't notice this was openGL.  Meshroom and cycles use CUDA.  Blender Render uses openGL.
Since you got through DepthMap, your card seems to compute CUDA, so it is strange it crashes at meshing.
First things first.  If your 1080ti can't use openGL there is already a serious issue with either the card or drivers.  Did you update to driver version 399.07? I had a driver from June that didn't work right that was fine after updating.

**Comment by bobokox** [comment]
Yep. Driver is the newest. I even deleted it totally and reinstall it, unfortunately it didn't solve the problem.
Just did a testing in Iray (NVidia Render Engine). I have the issue with the random blackscreen too, but the programm didn't crash the first 2 times. Unfortunately it did the third time. Same OpenGL error.
I hope I'll get response of the nvidia support tomorrow.
Thank you for your help!

**Comment by bobokox** [comment]
Hm. Didn't find a solution till now. I did a bios update, an update of the chipset drivers and a downgrade to an older nvida driver without success. Unfortunatly the nvidia support doesn't have more proposals, but they say that it couldn't be the hardware.

**Comment by bobokox** [comment]
I only have one monitor.
I've made some testscans. Most of the time (ca. 90%) it happens while FeatureExtraction. If it gets over this, DepthMap is the next critical point. The total reboot while Meshing only happened once till now.
I even did one succesful scan after I updated my chipset drivers yesterday. But the second attempt caused the error again, also all the attempts afterwards.
Just contacted the Gigabyte support, maybe they can give me some useful proposals.

Thank you for the help!
